scripts/config/conf.c

```diff
@@ -578,7 +578,7 @@ int main(int ac, char **av)
 				"*** Configuration file \"%s\" not found!\n"
 				"***\n"
 				"*** Please run some configurator (e.g. \"make oldconfig\" or\n"
-				"*** \"make menuconfig\" or \"make xconfig\").\n"
+				"*** \"make menuconfig\" or \"make xconfig\" or \"make nconfig\").\n"
```

Comment:
This message will not be shown under openwrt, and upstream does not have it, so you may drop it here if you wish.   If you try to run make without a config in openwrt, it will call menuconfig for you.